myapp/config/application.rb

```diff
@@ -22,5 +22,9 @@ class Application < Rails::Application
     config.generators do |generator|
       generator.fixture_replacement :factory_bot, suffix_factory: 'factory'
     end
+
+    config.i18n.default_locale = :en # change to :en back later so no need to rewrite test
```

Comment:
preferr to test in English 🙈 So I just check the display of Japanese version works fine~